about.html

```diff
@@ -0,0 +1,181 @@
+                    how they got to where they are,<br>
+                    and what motivation their experience might give us.
+                    Getting to know the big guns in town is very important.
+                    Throughout this session,
+                    we'll get to know some famous celebrities,
+                    how they got to where they are,<br>
+                    and what motivation their experience might give us.
+                    Getting to know the big guns in town is very important.
+                </p>
+            </div>
+            <div class="header2">
+                <p>Sometimes you may like to find out more about these celebs.
+                    If so, Please do not hesitate to contact us at
+                </p>
+                <a href="#">
+                    <p>@wils nkehebjlnjnkjw knflefwilj kjijeiikk</p>
+                </a>
+            </div>
+        </div>
+    </header>
+
+    <section class="main-program2">
+        <div class="main-section1">
+            <div class="head">
+                <h3>Main Program for the whole live session of this year</h3>
+                <div class="line"></div>
+            </div>
+
+            <div class="paragraph">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    Voluptatibus rerum asperiores ratione optio.
+                    Quas similique beatae vel repudiandae? Alias,
+                    fugiat. Atque fugit
+                    ex rem necessitatibus eius inventore.
+                    Lorem ipsum dolor sit amet consectetur
+                    adipisicing elit. Iusto, vero asperiores,
+                    natus cum cupiditate voluptate esse in ratione
+                    excepturi sint totam? Quibusdam culpa
+                    ipsa iusto iure rerum nisi autem deleniti.</p>
+            </div>
+
+            <div class="logo-container">
+                <img src="./Images/logo.jpg" alt="">
+            </div>
+        </div>
+    </section>
+
+    <hr>
+
+    <section class="main-program3">
+        <div class="head">
+            <h3>Main Program fpr the whole live session of this year</h3>
+            <div class="line"></div>
+        </div>
+        <div class="paragraph2">
+            <p>Lorem ipsum dolor sit amet,
+                consectetur adipisicing elit.
+                Ad et dicta commodi modi ipsum,
+                rem magni vero id, quidem nemo
+                earum beatae, quae vitae
+                accusamus nihil qui facere illo autem!
+            </p>
+        </div>
+        <div class="pic-container">
+            <div class="container1">
+                <h3>year 1</h3>
+                <p>Fulltime footballer expert</p>
+            </div>
+            <div class="container2">
+                <h3>year 1</h3>
+                <p>Fulltime footballer expert</p>
+            </div>
+        </div>
+    </section>
```

Comment:
This section can be improved a bit more, you can consider adding two images as you see in the given template instead of putting some texts that aren't styled. This would improve the looks of your page

<img width="962" alt="img22222" src="https://github.com/Wils-FOURR-speed/Capstone-1/assets/84930355/c1f9172a-126c-468b-9d59-6d2d9b593238">